build.psm1

```diff
@@ -100,12 +100,12 @@ function Get-EnvironmentInformation
     # Use the .NET Core APIs to determine the current platform.
     # If a runtime exception is thrown, we are on Windows PowerShell, not PowerShell Core,
```

Comment:
The line confuse me. I think it is better to re-write all the comment.

Reply:
I don't understand what's confusing about it. It's accurate.

Reply:
comment completely removed as it is no longer relevant.